src/oadp_resources/velero/backup.py

```diff
@@ -84,6 +86,10 @@ def failed(self):
     def deleting(self):
         return check_phase(self, self.BackupPhase.DELETING.value)
 
+    def done(self):
+        return self.failed_validation() or self.uploading_partial_failure() or \
```

Comment:
Here I think we should check its not in new, Inprogress phase. 
Later we can add the new phase waitingForPlugin phase. 

Let me know what you think ?

Reply:
@sbahar619  @PrasadJoshi12 
Lets go over the new flow together.
?

Reply:
Yea you are right, thanks!